MenuApplets/ClipboardHistory/main.cpp

```diff
@@ -73,6 +75,17 @@ int main(int argc, char* argv[])
         GUI::Clipboard::the().set_data(data_and_type.data, data_and_type.mime_type, data_and_type.metadata);
     };
 
+    auto delete_action = GUI::Action::create("Delete", { Mod_None, Key_Delete }, Gfx::Bitmap::load_from_file("/res/icons/16x16/stop-hand.png"), [&](const GUI::Action&) {
+        model->remove_item(table_view.selection().first().row());
+    });
+
+    auto entry_context_menu = GUI::Menu::construct();
+    entry_context_menu->add_action(delete_action);
+    table_view.on_context_menu_request = [&](const GUI::ModelIndex& index, const GUI::ContextMenuEvent& event) {
+        (void)index;
```

Comment:
you could just omit the parameter name:

```suggestion
    table_view.on_context_menu_request = [&](const GUI::ModelIndex&, const GUI::ContextMenuEvent& event) {
```